src/test/java/org/gridsuite/modification/server/modifications/VscModificationTest.java

```diff
@@ -262,6 +262,21 @@ public void testUnchangedHVDCangleDroopActivePowerControl() throws Exception {
         Assert.assertTrue(activePowerControl.isEnabled());
     }
 
+    //Test : p0 sould be required if drop is enabled
+    @Test
+    public void testHvdcAngleDroopActivePowerControlWithoutP0() {
```

Comment:
I think you should add one more test to cover this the scenario:
- p0 sould not be required if drop is disabled.

Reply:
done